product-mini/platforms/linux-sgx/enclave-sample/Makefile

```diff
@@ -12,6 +13,7 @@ SPEC_TEST ?= 0
 # These variables are automatically set by CMakeLists.txt
 SGX_IPFS = 0
 WAMR_BUILD_GLOBAL_HEAP_POOL = 0
+WASM_ENABLE_LIB_RATS = 1
```

Comment:
While Librats is disabled by default in CMake, this value is set to 1 and remains set to 1 even when the _iwasm_ is built without Librats. This explains why some unit tests fail.

Reply:
LIB_RATS has been set in L21:
```
LIB_RATS := $(shell if [ -d $(LIB_RATS_SRC) ]; then echo 1; else echo 0; fi)
```
But it checks whether there is `_deps/lib-rats` folder under `build` folder, seems that it is not a common method.

How about removing L21, and here use `LIB_RATS = 0` instead, and set it to 1/0 from CMakeLists.txt?